2025-06-26 10:59:16 error: StatusCode : 401, Message : Wrong authentication token
2025-06-26 10:59:36 error: StatusCode : 401, Message : Wrong authentication token
2025-06-26 11:00:27 error: StatusCode : 400, Message : is_deleted must be a boolean value
2025-06-26 11:00:32 error: StatusCode : 400, Message : is_deleted must be a boolean value
2025-06-26 11:17:56 error: StatusCode : 400, Message : is_deleted must be a boolean value
2025-06-26 11:18:00 error: StatusCode : 400, Message : is_deleted must be a boolean value
2025-06-26 11:18:02 error: StatusCode : 400, Message : is_deleted must be a boolean value
2025-06-26 11:18:02 error: StatusCode : 400, Message : is_deleted must be a boolean value
2025-06-26 11:18:04 error: StatusCode : 400, Message : is_deleted must be a boolean value
2025-06-26 11:18:05 error: StatusCode : 400, Message : is_deleted must be a boolean value
2025-06-26 11:18:32 error: StatusCode : 400, Message : Unexpected token '}', ..."ete_at":
}
" is not valid JSON
2025-06-26 11:18:48 error: StatusCode : 400, Message : is_deleted must be a boolean value
2025-06-26 11:47:21 error: StatusCode : 400, Message : Category ID is required
2025-06-26 11:50:39 error: StatusCode : 400, Message : Category ID is required
2025-06-26 11:52:47 error: StatusCode : 400, Message : Category ID is required
2025-06-26 11:54:06 error: StatusCode : 401, Message : Wrong authentication token
2025-06-26 11:56:32 error: StatusCode : 400, Message : Category ID is required
2025-06-26 12:04:55 error: StatusCode : 400, Message : Update data is empty
2025-06-26 12:06:05 error: StatusCode : 400, Message : Category ID is required
2025-06-26 12:12:59 error: StatusCode : 400, Message : Category ID is required
2025-06-26 12:16:27 error: StatusCode : 400, Message : Category ID is required
2025-06-26 12:21:46 error: StatusCode : 400, Message : Category ID is required
2025-06-26 14:00:32 error: StatusCode : 400, Message : client_id must be an integer number, editor_id must be an integer number, project_title should not be empty, project_title must be a string, Deadline must be a valid ISO 8601 date string, project_description should not be empty, project_description must be a string, Budget must be an integer number, each value in skills_required must be a string, skills_required should not be empty, skills_required must be an array, each value in reference_links must be a URL address, reference_links should not be empty, reference_links must be an array, additional_notes should not be empty, additional_notes must be a string, status must be an object, projects_type should not be empty, projects_type must be a string, project_format should not be empty, project_format must be a string, audio_voiceover should not be empty, audio_voiceover must be a string, video_length must be an integer number, preferred_video_style should not be empty, preferred_video_style must be a string, project_files must be an array, sample_project_file must be an array, show_all_files must be an array, created_by must be an integer number
2025-06-26 14:01:54 error: StatusCode : 400, Message : client_id must be an integer number, editor_id must be an integer number, project_title should not be empty, project_title must be a string, Deadline must be a valid ISO 8601 date string, project_description should not be empty, project_description must be a string, Budget must be an integer number, each value in skills_required must be a string, skills_required should not be empty, skills_required must be an array, each value in reference_links must be a URL address, reference_links should not be empty, reference_links must be an array, additional_notes should not be empty, additional_notes must be a string, status must be an object, projects_type should not be empty, projects_type must be a string, project_format should not be empty, project_format must be a string, audio_voiceover should not be empty, audio_voiceover must be a string, video_length must be an integer number, preferred_video_style should not be empty, preferred_video_style must be a string, project_files must be an array, sample_project_file must be an array, show_all_files must be an array, created_by must be an integer number
2025-06-26 14:03:58 error: StatusCode : 400, Message : client_id must be an integer number, editor_id must be an integer number, project_title should not be empty, project_title must be a string, Deadline must be a valid ISO 8601 date string, project_description should not be empty, project_description must be a string, Budget must be an integer number, each value in skills_required must be a string, skills_required should not be empty, skills_required must be an array, each value in reference_links must be a URL address, reference_links should not be empty, reference_links must be an array, additional_notes should not be empty, additional_notes must be a string, status must be an object, projects_type should not be empty, projects_type must be a string, project_format should not be empty, project_format must be a string, audio_voiceover should not be empty, audio_voiceover must be a string, video_length must be an integer number, preferred_video_style should not be empty, preferred_video_style must be a string, project_files must be an array, sample_project_file must be an array, show_all_files must be an array, created_by must be an integer number
2025-06-26 14:11:20 error: StatusCode : 404, Message : Authentication token missing
2025-06-26 17:45:39 error: StatusCode : 404, Message : Authentication token missing
2025-06-26 17:46:47 error: uncaughtException: Invalid Category
Error: Invalid Category
    at CategoryController.getprojectbycategory (D:\mmv1\mmv_freelance\backend\src\controllers\category.controller.ts:49:15)
    at D:\mmv1\mmv_freelance\backend\src\routes\categories.routes.ts:25:103
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:346:12)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:175:3)
2025-06-26 17:52:36 error: uncaughtException: Invalid Category
Error: Invalid Category
    at CategoryController.getprojectbycategory (D:\mmv1\mmv_freelance\backend\src\controllers\category.controller.ts:49:15)
    at D:\mmv1\mmv_freelance\backend\src\routes\categories.routes.ts:25:102
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:346:12)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:175:3)
2025-06-26 17:53:17 error: uncaughtException: Invalid Category
Error: Invalid Category
    at CategoryController.getprojectbycategory (D:\mmv1\mmv_freelance\backend\src\controllers\category.controller.ts:49:15)
    at D:\mmv1\mmv_freelance\backend\src\routes\categories.routes.ts:25:103
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:346:12)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:175:3)
2025-06-26 17:55:08 error: uncaughtException: Invalid Category
Error: Invalid Category
    at CategoryController.getprojectbycategory (D:\mmv1\mmv_freelance\backend\src\controllers\category.controller.ts:49:15)
    at D:\mmv1\mmv_freelance\backend\src\routes\categories.routes.ts:25:103
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:346:12)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:175:3)
2025-06-26 18:00:04 error: uncaughtException: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
error: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
    at Parser.parseErrorMessage (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:368:69)
    at Parser.handlePacket (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:187:21)
    at Parser.parse (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:102:30)
    at Socket.<anonymous> (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\index.ts:7:48)
    at Socket.emit (node:events:524:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Socket.Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-06-26 18:04:04 error: uncaughtException: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
error: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
    at Parser.parseErrorMessage (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:368:69)
    at Parser.handlePacket (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:187:21)
    at Parser.parse (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:102:30)
    at Socket.<anonymous> (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\index.ts:7:48)
    at Socket.emit (node:events:524:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Socket.Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-06-26 18:13:52 error: uncaughtException: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
error: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
    at Parser.parseErrorMessage (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:368:69)
    at Parser.handlePacket (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:187:21)
    at Parser.parse (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:102:30)
    at Socket.<anonymous> (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\index.ts:7:48)
    at Socket.emit (node:events:524:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Socket.Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-06-26 18:22:07 error: uncaughtException: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
error: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
    at Parser.parseErrorMessage (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:368:69)
    at Parser.handlePacket (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:187:21)
    at Parser.parse (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:102:30)
    at Socket.<anonymous> (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\index.ts:7:48)
    at Socket.emit (node:events:524:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Socket.Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-06-26 18:26:36 error: uncaughtException: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
error: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
    at Parser.parseErrorMessage (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:368:69)
    at Parser.handlePacket (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:187:21)
    at Parser.parse (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:102:30)
    at Socket.<anonymous> (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\index.ts:7:48)
    at Socket.emit (node:events:524:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Socket.Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-06-26 18:36:02 error: uncaughtException: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
error: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
    at Parser.parseErrorMessage (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:368:69)
    at Parser.handlePacket (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:187:21)
    at Parser.parse (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:102:30)
    at Socket.<anonymous> (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\index.ts:7:48)
    at Socket.emit (node:events:524:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Socket.Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-06-26 18:37:12 error: uncaughtException: Invalid Category
Error: Invalid Category
    at CategoryController.getprojectbycategory (D:\mmv1\mmv_freelance\backend\src\controllers\category.controller.ts:49:15)
    at D:\mmv1\mmv_freelance\backend\src\routes\categories.routes.ts:24:105
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:284:15
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:365:14)
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:421:3)
2025-06-26 18:39:58 error: uncaughtException: Invalid Category
Error: Invalid Category
    at CategoryController.getprojectbycategory (D:\mmv1\mmv_freelance\backend\src\controllers\category.controller.ts:49:15)
    at D:\mmv1\mmv_freelance\backend\src\routes\categories.routes.ts:24:114
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:284:15
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:365:14)
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:421:3)
2025-06-26 18:40:42 error: uncaughtException: Invalid Category
Error: Invalid Category
    at CategoryController.getprojectbycategory (D:\mmv1\mmv_freelance\backend\src\controllers\category.controller.ts:49:15)
    at D:\mmv1\mmv_freelance\backend\src\routes\categories.routes.ts:24:104
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:284:15
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:365:14)
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:421:3)
2025-06-26 19:02:07 error: uncaughtException: Invalid Category
Error: Invalid Category
    at CategoryController.getprojectbycategory (D:\mmv1\mmv_freelance\backend\src\controllers\category.controller.ts:49:15)
    at D:\mmv1\mmv_freelance\backend\src\routes\categories.routes.ts:24:115
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:284:15
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:365:14)
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:421:3)
2025-06-26 19:05:27 error: uncaughtException: Invalid Category
Error: Invalid Category
    at CategoryController.getprojectbycategory (D:\mmv1\mmv_freelance\backend\src\controllers\category.controller.ts:49:15)
    at D:\mmv1\mmv_freelance\backend\src\routes\categories.routes.ts:24:110
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:284:15
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:365:14)
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:421:3)
2025-06-26 19:05:54 error: uncaughtException: Invalid Category
Error: Invalid Category
    at CategoryController.getprojectbycategory (D:\mmv1\mmv_freelance\backend\src\controllers\category.controller.ts:49:15)
    at D:\mmv1\mmv_freelance\backend\src\routes\categories.routes.ts:24:106
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:284:15
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:365:14)
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:421:3)
2025-06-26 19:19:44 error: uncaughtException: Invalid Category
Error: Invalid Category
    at CategoryController.getprojectbycategory (D:\mmv1\mmv_freelance\backend\src\controllers\category.controller.ts:49:15)
    at D:\mmv1\mmv_freelance\backend\src\routes\categories.routes.ts:24:107
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:284:15
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:365:14)
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:421:3)
2025-06-26 19:44:04 error: uncaughtException: Invalid Category
Error: Invalid Category
    at CategoryController.getprojectbycategory (D:\mmv1\mmv_freelance\backend\src\controllers\category.controller.ts:49:15)
    at D:\mmv1\mmv_freelance\backend\src\routes\categories.routes.ts:24:105
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:284:15
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:365:14)
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:421:3)
2025-06-26 19:55:18 error: uncaughtException: Invalid Category
Error: Invalid Category
    at CategoryController.getprojectbycategory (D:\mmv1\mmv_freelance\backend\src\controllers\category.controller.ts:49:15)
    at D:\mmv1\mmv_freelance\backend\src\routes\categories.routes.ts:24:105
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\layer.js:95:5)
    at D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:284:15
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:365:14)
    at param (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (D:\mmv1\mmv_freelance\backend\node_modules\express\lib\router\index.js:421:3)
2025-06-26 20:07:28 error: uncaughtException: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
error: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
    at Parser.parseErrorMessage (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:368:69)
    at Parser.handlePacket (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:187:21)
    at Parser.parse (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:102:30)
    at Socket.<anonymous> (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\index.ts:7:48)
    at Socket.emit (node:events:524:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Socket.Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-06-26 20:10:16 error: uncaughtException: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
error: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
    at Parser.parseErrorMessage (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:368:69)
    at Parser.handlePacket (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:187:21)
    at Parser.parse (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:102:30)
    at Socket.<anonymous> (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\index.ts:7:48)
    at Socket.emit (node:events:524:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Socket.Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-06-26 20:37:57 error: uncaughtException: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."project_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.project_id does not exist
error: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."project_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.project_id does not exist
    at Parser.parseErrorMessage (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:368:69)
    at Parser.handlePacket (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:187:21)
    at Parser.parse (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:102:30)
    at Socket.<anonymous> (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\index.ts:7:48)
    at Socket.emit (node:events:524:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Socket.Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-06-26 20:41:38 error: uncaughtException: select "category".*, "projects_task".* from "category" inner join "project_categories" on "category"."category_id" = "project_categories"."category_id" inner join "projects_task" on "project_categories"."project_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - relation "project_categories" does not exist
error: select "category".*, "projects_task".* from "category" inner join "project_categories" on "category"."category_id" = "project_categories"."category_id" inner join "projects_task" on "project_categories"."project_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - relation "project_categories" does not exist
    at Parser.parseErrorMessage (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:368:69)
    at Parser.handlePacket (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:187:21)
    at Parser.parse (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:102:30)
    at Socket.<anonymous> (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\index.ts:7:48)
    at Socket.emit (node:events:524:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Socket.Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-06-26 20:43:36 error: uncaughtException: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."project_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.project_id does not exist
error: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."project_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.project_id does not exist
    at Parser.parseErrorMessage (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:368:69)
    at Parser.handlePacket (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:187:21)
    at Parser.parse (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:102:30)
    at Socket.<anonymous> (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\index.ts:7:48)
    at Socket.emit (node:events:524:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Socket.Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-06-26 20:45:14 error: uncaughtException: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."project_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.project_id does not exist
error: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."project_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.project_id does not exist
    at Parser.parseErrorMessage (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:368:69)
    at Parser.handlePacket (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:187:21)
    at Parser.parse (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:102:30)
    at Socket.<anonymous> (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\index.ts:7:48)
    at Socket.emit (node:events:524:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Socket.Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-06-26 20:45:45 error: uncaughtException: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."project_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.project_id does not exist
error: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."project_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.project_id does not exist
    at Parser.parseErrorMessage (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:368:69)
    at Parser.handlePacket (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:187:21)
    at Parser.parse (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:102:30)
    at Socket.<anonymous> (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\index.ts:7:48)
    at Socket.emit (node:events:524:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Socket.Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-06-26 20:49:52 error: uncaughtException: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."project_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.project_id does not exist
error: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."project_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.project_id does not exist
    at Parser.parseErrorMessage (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:368:69)
    at Parser.handlePacket (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:187:21)
    at Parser.parse (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:102:30)
    at Socket.<anonymous> (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\index.ts:7:48)
    at Socket.emit (node:events:524:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Socket.Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
2025-06-26 20:51:49 error: uncaughtException: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
error: select "category".*, "projects_task".* from "category" inner join "projects_task" on "category"."projects_task_id" = "projects_task"."projects_task_id" where "category"."category_id" = $1 and "category"."is_deleted" = $2 order by "category"."created_at" desc - column category.projects_task_id does not exist
    at Parser.parseErrorMessage (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:368:69)
    at Parser.handlePacket (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:187:21)
    at Parser.parse (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\parser.ts:102:30)
    at Socket.<anonymous> (D:\mmv1\mmv_freelance\backend\node_modules\pg-protocol\src\index.ts:7:48)
    at Socket.emit (node:events:524:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Socket.Readable.push (node:internal/streams/readable:392:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:189:23)
